scripts/graphql-schema-validate.js

```diff
@@ -0,0 +1,38 @@
+import fs from 'fs';
+import graphql from 'graphql';
+
+// Validate JSON schema
+const raw = fs.readFileSync('graphql/schemas/schema.gql', 'utf8');
```

Comment:
Please rename the file to `graphql-validate-tests.js`. Or merge it with the file `graphql-validate.js`. IMO either is fine.